lib/SILOptimizer/Mandatory/TFDifferentiation.cpp

```diff
@@ -787,6 +1101,10 @@ static void debugDump(T &v) {
         << v << "==== END DEBUG DUMP ====\n");
 }
 
+static inline raw_ostream &getDebugStream() {
```

Comment:
This is a bit weird: it doesn't return a stream, it returns a stream and prints stuff.

It would be great to rename this, e.g. getADDebugStream, but also please move it to the top of the file and drop the forward include.